doc/submitter_guide.md

```diff
@@ -60,19 +60,18 @@ dashboard](https://kcidb.kernelci.org/).
 2\. Install KCIDB
 -----------------
 
-Pick a KCIDB [release][releases]. Latest would normally be best.
 
-Let's say you picked v9. Run this:
+This will install the latest version of the package from the main branch of the kcidb repository.
```

Comment:
We need to be clear that picking up latest `main` is OK here, and explain the development philosophy. Something like this would do, but feel free to expand/modify, if you have better ideas:

> KCIDB employs continuous integration and delivery, and aims to keep the code working at all times. Please install the latest version from GitHub:

Reply:
Okay